app/uk/gov/hmrc/eusubsidycompliancefrontend/models/types/package.scala

```diff
@@ -74,7 +74,13 @@ package object types extends SimpleJson {
   object EORI
       extends RegexValidatedString(
         """^(GB|XI)[0-9]{12,15}$"""
-      )
+      ) {
+    private val EoriPrefix = "GB"
+
+    def withPrefix(eori: String): String =
+      if (eori.startsWith(EoriPrefix)) eori
```

Comment:
The regex above seems to suggest it can also start with XI? Is that a consideration?

Reply:
`withPrefix` is specifically for GB eoris. I'll rename to `withGBPrefix` to make this a little clearer. I don't think we'll ever allow entry of `XI` prefixed EORIs but since I'm unsure I'll leave the regex as is for now.